### ISSUE TITLE
X3D_m pretrained model from pytorch cannot be exported to ONNX

### ISSUE THREAD

**Issue by callbarian** [pytorch/pytorch#61008]
If you have a question or would like help and support, please ask at our
[forums](https://discuss.pytorch.org/).

If you are submitting a feature request, please preface the title with [feature request].
If you are submitting a bug report, please fill in the following details.

## Issue description
ONNX export failed: Couldn't export Python operator SwishFunction

Provide a short description.
Pytorch X3D_m pretrained model cannot be exported to ONNX.

## Code example
model_name = "x3d_m"
model = torch.hub.load('facebookresearch/pytorchvideo',model_name, pretrained=True)
...
in_names = ['input']
out_names = ['output']
torch.onnx.export(model, 
                            inputs[None,...], 
                            os.path.join(onnx_folder,model_name+'_opset11.onnx'), 
                            input_names=in_names,
                            output_names=out_names,
                            opset_version=11,
                            verbose=True,
                            dynamic_axes={'input': {0 : 'batch_size'},
                                                        'output': {0 : 'batch_size'}})

Please try to provide a minimal example to repro the bug.
Error messages and stack traces are also helpful.

## System Info
Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

- PyTorch or Caffe2: Pytorch
- How you installed PyTorch (conda, pip, source): conda
- Build command you used (if compiling from source):
- OS: Ubuntu 18.04
- PyTorch version: 1.7.1
- Python version: 3.8
- CUDA/cuDNN version: 11.0
- GPU models and configuration: RTX 3090
- GCC version (if compiling from source):
- CMake version:
- Versions of any other relevant libraries: PytorchVideo 0.1.2 , Torchvision 0.8.2

**Comment by callbarian** [comment]
The SwishFunction could be altered to non memory efficient SwishFunction which is (x * torch.sigmoid(x)).
Instead of using torch.autograd.Function use nn.Module to 
and remove ctx.saved_variables[0]

I have changed Swish class in the pytorchvideo/layers/swish.py 
to

class Swish(nn.Module):
    """
    Wrapper for the Swish activation function.
    """
    # @staticmethod
    def forward(self, x):
        result = x * torch.sigmoid(x)
        # ctx.save_for_backward(x)
        return result
        # return SwishFunction.apply(x)
